Reflection_Engine/Query/Description.cs

```diff
@@ -185,6 +185,43 @@ public static string IDescription(this object item)
                 return "";
         }
 
+        /***************************************************/
+
+        [Description("Return the custom description of a classification attribute.")]
+        [Input("classification", "Classification attribute to be queried for description.")]
+        public static string IDescription(this InputClassificationAttribute classification)
+        {
+            return Description(classification as dynamic);
+        }
+
+        /***************************************************/
+
+        [Description("Return the custom description of a quantity attribute.")]
+        [Input("quantity", "Quantity attribute to be queried for description.")]
+        public static string Description(this QuantityAttribute quantity)
+        {
+            return "This is a " + quantity.GetType().Name + " [" + quantity.SIUnit + "]";
+        }
+
+        /***************************************************/
+
+        [Description("Return the custom description of a folder path attribute.")]
```

Comment:
"custom"

"This is a folder path."

One of these is not like the other... :wink: can we update the description to be accurate to the code? :smile:

Reply:
_custom_ keyword removed 🙈